sagar/data/Lab_1sagar.ipynb

```diff
@@ -0,0 +1,1314 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    populations = [4.4, 3.9, 7.8, 3.5, 6.9]
</pre><p>What us the unit here? Millions? Thousands?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/suparthghimire/data-analysis-and-visualization-bca-2025/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>